Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -37,6 +37,9 @@ public class CurrentChatUserController: DataController, DelegateCallable, DataSt
         return _basePublishers as? BasePublishers ?? .init(controller: self)
     }
 
+    /// The observer for the active live location messages.
```

Comment:
what about the state layer objects? Shouldn't we also expose the location feature?

Reply:
Yeah @laevandus also mentioned this. it is better we create a separate PR, since this is one is already way to big. I will create a ticket for this